boards/BMS/Inc/App/App_FaultManagerConfig.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+#define FSM_CAN App_CanRx_FSM_ERRORS
+#define DCM_CAN App_CanRx_DCM_ERRORS
+#define PDM_CAN App_CanRx_PDM_ERRORS
+#define DIM_CAN App_CanRx_DIM_ERRORS
+#define BMS_CAN App_CanTx
```

Comment:
Why is the `BMS_CAN` different from all the others here?

Reply:
It's because BMS_ERRORS is TX from the point of view of BMS board.

If I were writing the config header for DCM, it would look like

```
#define FSM_CAN App_CanRx_FSM_ERRORS
#define DCM_CAN App_CanTx
#define PDM_CAN App_CanRx_PDM_ERRORS
#define DIM_CAN App_CanRx_DIM_ERRORS
#define BMS_CAN App_CanRx_BMS_ERRORS
```